lms/static/sass/base/_animations.scss

```diff
@@ -279,3 +279,138 @@
     opacity: 0;
   }
 }
+
+
+// +Fade In - Extend
```

Comment:
Thanks for adding these. Just to be sure, can you confirm that these are not already being imported into the LMS style by another Sass partial?